tests/functional/api/test_repository.py

```diff
@@ -46,7 +46,7 @@ def test_repository_files(project):
     blame = project.files.blame(file_path="README.rst", ref="main")
     assert blame
 
-    raw_file = project.files.raw(file_path="README.rst", ref="main")
+    raw_file = project.files.raw(file_path="README.rst")
     assert os.fsdecode(raw_file) == "Initial content"
 
```

Comment:
As you can see we now don't cover that path, what if we just add a simple assert like this. Not sure if you want to be more elaborate in testing:

```suggestion
    raw_file = project.files.raw(file_path="README.rst")
    assert os.fsdecode(raw_file) == "Initial content"

    raw_file_from_ref = project.files.raw(file_path="README.rst", ref="main")
    assert raw_file == raw_file_from_ref
```

Reply:
Ok, I see you've already done something similar, never mind!

Reply:
Had exactly the same thought!